lib/spack/docs/conf.py

```diff
@@ -68,7 +68,6 @@
 # Without this, the API Docs will never actually update
 #
 apidoc_args = [
-    '--force',         # Older versions of Sphinx ignore the first argument
```

Comment:
This bug should be fixed in Sphinx 1.8+, see https://github.com/sphinx-doc/sphinx/pull/3668